src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -123,6 +122,16 @@ def download_manifests(client: InventoryClient, cluster_id: str, output_folder:
     client.download_manifests(cluster_id, manifests_path)
 
 
+def merge_events(event_paths: list[str]) -> str:
+    events = []
+
+    for event_path in event_paths:
+        with open(event_path, "rt") as event_file:
+            events.extend(json.loads(event_file.read()))
```

Comment:
Maybe `SuppressAndLog` here so we won't lose all the events due to single event failure? 
Note: not sure if this is a valid case

Reply:
It extends the list for each source (cluster and each infraenv object attached to it) and not for every event

Since test-infra created the events file a few lines before, I don't expect it to actually fail (if ``client.download_infraenv_events`` fails, then the addition to ``event_files`` wouldn't happen as it's one line after)